doc/base/classes.xml

```diff
@@ -45707,6 +45729,7 @@
 			<argument index="0" name="enable" type="bool">
 			</argument>
 			<description>
+				Set whether this node sends notification that its local transformation changed. False by default.
```

Comment:
I would remove 'local' word from this sentence

Reply:
Also "false by default" indicate a little that there is default parameter in this function with value=false. I would write that "Spatial will not propagate this by default." or something similar

Reply:
Done. updated other occurrences as well.